docs/zed/api.md

```diff
@@ -200,42 +462,26 @@ data: {"pool_id": "1sMDXpVwqxm36Rc2vfrmgizc3jz", "commit_id": "1tisISpHoWI7MAZdF
 
 event: pool-delete
 data: {"pool_id": "1sMDXpVwqxm36Rc2vfrmgizc3jz"}
-
-```
-
-### Index Rules
-
-#### Create Index Rule
-
-Creates an index rule for a specified field.
-
 ```
-POST /index
-```
-
-#### Delete Index Rule
 
-Deletes the specified index rule. Any created object indexes will persist.
-
-```
-DELETE /index
-```
+---
 
 ## Media Types
 
-For responses content types, the service can handle a variety of formats. To
-receive responses in the desired format, include the mime type of the format in
-the requests ACCEPT HTTP header.
+For response content types, the service can produce a variety of formats. To
+receive responses in the desired format, include the MIME type of the format in
+the request's Accept HTTP header.
 
-If the ACCEPT header is not specified, the service will return json as default.
+If the Accept header is not specified, the service will return JSON as the
+default response format for the endpoints described above, with the exception
+of the [query](#query) endpoint that returns ZSON by default.
```

Comment:
I also pre-cleared this one with @mattnibs since I wasn't sure if the ZSON default returned by `/query` was a bug or a feature, and the consensus was to document it as expected.

Reply:
Thought about this a little more. It's weird to have an exception like this. My proposal is this: ZSON by default on every endpoint. If the content-type is `application/x-www-form-urlencoded` (the default for curl) it should return JSON. @philrz what do you think about this?